content/applications/marketing/social_marketing/essentials/social_essentials.rst

```diff
@@ -0,0 +1,355 @@
+   will not appear as an option on the post template.
+
+Next, there's the :guilabel:`Message` field. This is where the main content of the post is created.
+
+Type the desired message for the post in this field. To the right, as the :guilabel:`Message` field
+is populated, Odoo displays visual samples of how the post will look on all the previously selected
+social media accounts from the :guilabel:`Post on` field above.
+
+.. tip::
+   Emojis can also be added directly to the text in the :guilabel:`Message` field. Just click the
+   "smiley face" icon, located on the line of the :guilabel:`Message` field to the far right.
+   Clicking that "smiley face" icon reveals a drop-down containing numerous emojis to choose from.
+
+If images are to be used in the post, click the :guilabel:`ATTACH IMAGES` link beneath the
+:guilabel:`Message` field, and Odoo reveals a pop-up window. In this pop-up, the desired image must
+be chosen, and then uploaded.
+
+A preview of the entire post, text and image (if applicable), is instantly displayed in the visual
+preview of the post.
+
+Next, there's the option to attach this post to a specific marketing campaign in the database in
+the :guilabel:`Campaign` field. Click the blank line next to :guilabel:`Campaign` to reveal the
+previously configured campaigns to choose from.
+
+.. tip::
+   A new campaign can be created on-the-fly, as well, by typing the name of the new campaign on the
+   blank :guilabel:`Campaign` field, and selecting :guilabel:`Create` from the drop-down. Or, select
+   :guilabel:`Create and edit` from the drop-down to further customize that newly-created campaign.
+
+.. note::
+   A social post does *not* need to be attached to a campaign.
+
+Then, in the :guilabel:`When` field, choose either :guilabel:`Send Now` to have Odoo publish the
+post immediately, or :guilabel:`Schedule later` to have Odoo publish the post at a later date and
+time.
+
+If :guilabel:`Schedule later` is selected, Odoo reveals a new field beneath it (the
+:guilabel:`Scheduled post date` field). Clicking that empty field reveals a pop-up calendar, in
+which a future date and time is designated. At which time, Odoo will promptly publish the post on
+the pre-determined social media accounts.
+
+Click on the desired date to schedule the post for that day. Then, either select and customize
+the default time in the :guilabel:`Scheduled post date` field manually. Or, adjust the desired post
+time, by clicking the **clock icon** located on the calendar pop-up, and choose the desired time for
+Odoo to publish this post on that future date.
+
+If scheduling a post, remember to hit :guilabel:`Schedule` in the upper left of the post template.
+Doing so, locks in that specific date/time for Odoo to send the post, and it changes the status of
+the post to :guilabel:`Scheduled`, which will be discussed later on in this documentation, in the
+:guilabel:`Social post status bar` section.
```

Comment:
The last part of the sentence mixes up the subject(s) of the sentence, so we can remove it.

```suggestion
the post to :guilabel:`Scheduled`.
```